languages/translations/en.json

```diff
@@ -254,7 +254,7 @@
                 "name": "Take the ramp on the right onto {way_name}",
                 "destination": "Take the ramp on the right towards {destination}",
                 "exit": "Take exit {exit}",
-                "exit_destination": "Take exit {exit} towards {destination}"
+                "exit_destination": "Take exit {exit} on the right towards {destination}"
```

Comment:
I'd love to remove the direction from instructions, except when we have no way/destination/exit at all: `Take the ramp on the right`.